src/Input.php

```diff
@@ -130,11 +132,7 @@ public function __get($name)
             return $this->inputs[$name];
         }
 
-        $trace = debug_backtrace();
-        trigger_error(
-            'Undefined property via __get(): ' . $name . ' in ' . $trace[0]['file'] . ' on line ' . $trace[0]['line'],
-            E_USER_NOTICE
-        );
+        throw new Exception();
```

Comment:
```suggestion
        throw new Exception('Undefined property via __get(): ' . $name);
```
Shouldn't we add the message?